.github/workflows/cd.yml

```diff
@@ -0,0 +1,64 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
maybe if we comment this part the job doesn't run by default (what we want) and in the docs we can clarify that they need to uncomment it if they are willing to deploy to AWS?

Reply:
Similar to what we did here: https://github.com/rootstrap/rails_api_base/blob/f999c2f5d9fa252db95923036389e8ee2eead578/.github/workflows/update_knapsack_report.yml#L3-L8

And we can even add worfklow_dispatch so that it can be manually triggered